### DISCUSSION TITLE
Installation Fail

### DISCUSSION THREAD

**Discussion by jsavary87** [Security-Onion-Solutions/securityonion#2358] [General] [answered]
Hello,

Installation fails due to so-acng signature error. Please help.

Update: it says there is an error downloading the so-acng. Warning: unsafe ownership of homdir    gpg signature made is BAD

**Comment by TOoSmOotH** [comment, marked as answer]
Sounds like it can't get to AWS to download the gpg signature. Have you tried using the ISO?

**Reply by jsavary87** [reply]
According to this person: https://github.com/Security-Onion-Solutions/securityonion/discussions/2285#discussioncomment-198643

it would appear that there was a docker update or something that changed their network causing these issues :(

Also, I have created a VM and dropped the ISO into it but that fails as well. I cant see where it fails because the log populates too fast and then wipes and boots back into installation menu immediately. No way to tell where it fails so thats why I made a ubuntu (18 version) and am trying to install on top of that. 

Thank you for your time though! I do greatly appreciate the assistance!

**Reply by jsavary87** [reply]
Also, does 2.3.10 have to be installed with CentOS as the underlying OS? or can it be Ubuntu like in previous versions? The reason I ask is because VirtualBox doesnt have CentOS as an option.

**Reply by dougburks** [reply]
VirtualBox does indeed support CentOS, so you should be able to perform any of our installation methods in VirtualBox:
- installing our Security Onion ISO image
- manually installing CentOS 7 and then installing our components on top
- manually installing Ubuntu 18.04 and then installing our components on top

For more information, please see:
https://docs.securityonion.net/en/2.3/virtualbox.html
https://docs.securityonion.net/en/2.3/installation.html
